_opensearch/rest-api/cat/cat-segments.md

```diff
@@ -46,6 +46,7 @@ In addition to the [common URL parameters]({{site.url}}{{site.baseurl}}/opensear
 Parameter | Type | Description
 :--- | :--- | :---
 bytes | Byte size | Specify the units for byte size. For example, `7kb` or `6gb`. For more information, see [Supported units]({{site.url}}{{site.baseurl}}/opensearch/units/)..
+master_timeout | Time | The amount of time to wait for a connection to the master node. Default is 30 seconds.
```

Comment:
What is the unit for this setting? Is it in seconds or do I need to specify the units like for `bytes`? The doc should clarify.

Reply:
Good advice 😄. 
The unit has to be specified, such as (`1h` stands for 1 hour, `2m` stands for 2 minutes). But probably there should be an additional page to explain the units, and put a hyperlink near here.

Reply:
@tlfeng and @dblock: Since your suggestion covers more than this instance of the `master_timeout`, would you be ok with us filing an issue to modify the `master_timeout` description to explain that units need to be specified?

Reply:
Yeah, I don't think the `Time` unit have to be explained in this Pull Request, because it will be a long story.